connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -48,6 +48,7 @@ message Expression {
     CommonInlineUserDefinedFunction common_inline_user_defined_function = 15;
     CallFunction call_function = 16;
     NamedArgumentExpression named_argument_expression = 17;
+    TypedAggregateExpression typed_aggregate_expression = 18;
```

Comment:
Why do we need a separate expression? It is just an invocation with a wild-card.

Reply:
With this new expression it is very easy to identify this `toColumn` usage and transform it into a query plan which 1:1 matches a plan produced by the non-Connect version of Spark.

> It is just an invocation with a wild-card

Could you elaborate? Do you mean something like...
`new UserDefinedFunction(...).apply(col("*")).expr` then create a TypedColumn from it?
I tried this one but it seems more modification is needed to get rid of the Star.